Improve documentation for JS DCE (#1861)
JetBrains/kotlin-web-site: pages/docs/reference/javascript-dce.md

  adapters for DOM, and so on. All of this functionality would require about 1.3 MB as a JavaScript file. A simple "Hello, world" application only requires
  console routines, which is only few kilobytes for the entire file.

> > The Kotlin/JS Gradle plugin handles DCE automatically when you build a production bundle, for example by using the `browserProductionWebpack` task. Development bundling tasks (like `browserDevelopmentWebpack`) don't include DCE.

Reviewer: I don't think this should be a note. It's an important part of the general explanation. Notes are usually for some side additions that don't fit into the common flow.

Author: Thanks, I have removed the note tag from this, and added some bold emphasis to show the two differing cases instead.

list the declarations as arguments of the `keep` function. An argument must be the declaration's fully qualified name
with the module name as a prefix: `moduleName.dot.separated.package.name.declarationName`

> Unless specified otherwise, the names of functions and modules can be [mangled](js-to-kotlin-interop.html#jsname-annotation) in the generated JavaScript code. To keep such functions from elimination, use the mangled names in the `keep` arguments as they appear in the generated JavaScript code.
{:.note}

<div class="sample" markdown="1" mode="groovy" theme="idea">

```groovy
kotlin {
    js {
        browser {
            dceTask {
                keep("myKotlinJSModule.org.example.getName", "myKotlinJSModule.org.example.User" )
            }
            binaries.executable()
        }
    }
}
```
</div>

If you want to keep a whole package or module from elimination, you can use its fully qualified name as it appears in the generated JavaScript code.

> > Keeping whole packages or modules from elimination can prevent DCE from removing many declarations that otherwise would have been removed. Because of this, it is preferable to select individual declarations which should be excluded from DCE one by one.

Reviewer: ```suggestion
> Keeping whole packages or modules from elimination can prevent DCE from removing many unused declarations. Because of this, it is preferable to select individual declarations which should be excluded from DCE one by one.
```
If we shorten the sentence like this, will it keep the meaning?

Author: Yup, that still sounds good to me!
